RegexandParsing/Validatingphonenumbers.py

```diff
@@ -4,19 +4,20 @@
 Domain    : Python
 Author    : Ahmedur Rahman Shovon
 Created   : 15 July 2016
+Updated   : 3 April 2021
 Problem   : https://www.hackerrank.com/challenges/validating-the-phone-number/problem
 '''
 # Enter your code here. Read input from STDIN. Print output to STDOUT
-n=int(raw_input())
+n=int(input())
```

Comment:
@HighnessAtharva ,

The following code is using regex. Please update the PR with this:

```
from re import compile, match


n = int(input())
for _ in range(n):
    phone_number = input()
    condition = compile(r'^[7-9]\d{9}$')
    if bool(match(condition, phone_number)):
        print("YES")
    else:
        print("NO")
```

Reply:
Done.